### PR TITLE
ESLint rules added   GsSoC'23

### PR DESCRIPTION
**Pull request by Nikhil1017**
As per issue #410

### CODE REVIEW

**Comment by Nikhil1017** [comment]
@PiyushKalyanpy

**Comment by Nikhil1017** [comment]
It has successfully deployed now

File: .eslintrc.json [hunk 1]
```diff
@@ -1,3 +1,10 @@
 {
-  "extends": "next/core-web-vitals"
+  "extends": ["next", "next/core-web-vitals", "eslint:recommended"],
+  "globals": {
+    "React": "readonly",
+    "NodeJS": "readonly"
+  },
+  "rules": {
+    "no-unused-vars": "warn"
```
Comment (lalitmee) [inline .eslintrc.json:8]: @PiyushKalyanpy and @Nikhil1017 please check these rules too.
```js
  rules: {
    'prettier/prettier': ['error', prettierOptions],
    'arrow-body-style': [2, 'as-needed'],
    'class-methods-use-this': 0,
    'import/imports-first': 0,
    'import/newline-after-import': 0,
    'import/no-dynamic-require': 0,
    'import/no-extraneous-dependencies': 0,
    'import/no-named-as-default': 0,
    'import/no-webpack-loader-syntax': 0,
    'import/prefer-default-export': 0,
    indent: [2, 2, { SwitchCase: 1 }],
    'max-len': 0,
    'newline-per-chained-call': 0,
    'no-confusing-arrow': 0,
    'no-console': 1,
    'no-unused-vars': 2,
    'no-use-before-define': [
      'error',
      { functions: false, classes: false, variables: true },
    ],
    'prefer-template': 2,
    'require-yield': 0,
  },

```

File: components/RightSideBar/Notes.tsx [hunk 2]
```diff
@@ -1,14 +1,14 @@
-import React from "react";
-import Image from "next/image";
-type Notes = {
+import React from 'react';
+import Image from 'next/image';
+type NotesCard = {
```
Comment (lalitmee) [inline components/RightSideBar/Notes.tsx:3]: Leave an empty line after imports

**Review by lalitmee — CHANGES_REQUESTED** [review]

**Comment by Nikhil1017** [comment]
@lalitmee thanks for your suggestions, I have incorporated the rules and put blank line near import, you can check now

**Comment by Nikhil1017** [comment]
@Vaishnavi2701-mk

**Review by lalitmee — APPROVED** [review]

**Comment by Nikhil1017** [comment]
@lalitmee conflicts resolved

**Review by lalitmee — APPROVED** [review]

**Comment by Nikhil1017** [comment]
@PiyushKalyanpy

**Review by PiyushKalyanpy — APPROVED** [review]

**Comment by PiyushKalyanpy** [comment]
Hey @Nikhil1017 , Thank you 🎉for your valuable contribution.

**Comment by Nikhil1017** [comment]
thanks @PiyushKalyanpy

**Pull request merged**